coala_quickstart/coala_quickstart.py

```diff
@@ -57,13 +58,18 @@ def _get_arg_parser():
     return arg_parser
 
 
+def signal_handler(self, cls):
+    sys.exit(0)
+
+
```

Comment:
no need of two blank lines.

Reply:
the newlines are fine

Reply:
although I didn't mean for you to add both self and cls

Reply:
both are actually the same, with different names

Reply:
btw if you are using this function only once, no need of even creating one

Reply:
should i use lambda function

Reply:
why? just perform system exit when you have to without a function

Reply:
It is difficult to know when a person will system exit.

Reply:
So it's kind of better to initialise exception handler first

Reply:
hmmm... frankly speaking I don't understand what you are trying to do over here 🤪 but adding those two parameters without knowing why its working that way is definitely wrong

Reply:
i am trying to capture the exception globally.

Reply:
i will fix remove the unnecessary parameters.